src/test/java/step1/StringTest.java

```diff
@@ -0,0 +1,35 @@
+package step1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+public class StringTest {
+    @Test
+    @DisplayName("요구사항1: 문자열 split")
+    public void split() {
+        String target = "1,2";
+        String[] splitResults = target.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?